### PR TITLE
Remove horizontal scroll bar

### LINKED ISSUE #127: A horizontal scroll bar appears on smaller screens
## Summary
A horizon scroll bar appears on smaller screens - [microstack.run](microstack.run)

## Current and expected result

A horizon scroll bar appears on smaller screens. -  Make it responsive so the scroll bar doesn't appear  

## Screenshot

image

## Browser details

Refer to this part in this [website](https://jaas.ai/) 
image

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done
- Replaced the `<pre>` and `<code>` element with `<div>` with styles
- Added `_row-cli.scss` file in static/sass for styles

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8039/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Please check if the code snippet's animation is working well and it's responsive according to screen size
## Issue / Card

Fixes #127 

## Screenshots

image

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -40,7 +40,7 @@ <h1> OpenStack for the edge, <br/> micro clouds and developers </h1>
         <h2>Straightforward installation</h2>
         <h3 class="p-heading--4">Fully functional OpenStack in 2 commands and ~20 minutes.</h3>
       </div>
-      <pre class="p-heading--two"><code>$ sudo <span class="typer" id="main" data-words="snap install microstack --beta --devmode,microstack init --auto --control" data-delay="50" data-deleteDelay="5000" data-colors="#111111"></span></code></pre>
+      <div class="row-cli__commands" style="margin-bottom: 1rem;">$ sudo<span class="row-cli__commands-list typer" id="main" data-words="snap install microstack --beta --devmode,microstack init --auto --control" data-delay="50" data-deleteDelay="5000" data-colors="#111111"></span></div>
```
Comment (bethcollins92) [inline templates/index.html:43]: Could you not add the margin bottom style to the `row-cli__commands` class?

**Review by bethcollins92 — COMMENTED** [review]
I checked on an Iphone X screen and there was an ellipses in the command: 

Screenshot 2021-09-07 at 13 41 50

But this ellipses also happens on juju. @anthonydillon do you know if this is expected behaviour or if this is a bug?

Screenshot 2021-09-07 at 13 42 45

File: templates/index.html (hunk 1 above)
Comment (minkyngkm) [reply templates/index.html]: Fixed! :)

Comment (minkyngkm) [reply templates/index.html]: Fixed 👍

**Comment by anthonydillon** [comment]
I see the same on my phone. Difficult to know really. Hiding any part of the command is bad. Maybe the command should wrap in small screens. Unless we can set the font size based on the `vw`.

**Comment by clagom** [comment]
+1, wrap the command line seems like a good option for small screens

**Comment by bethcollins92** [comment]
Thanks for the feedback @anthonydillon and @clagom, let me know when it's ready for a review again @minkyngkm (happy to pair if needed). I'll create an issue on juju.is too.

**Comment by minkyngkm** [comment]
Thank you @bethcollins92, @anthonydillon and @clagom 🙏🏻 . I have updated it. Hopefully it looks better! - [demo](https://microstack-run-128.demos.haus/)

For your reference, I updated the `line-height` of the command line from `1rem` to `1.2rem` in case of the second line appearing on small screens. Please let me know if it's okay Beth :)

**Review by bethcollins92 — APPROVED** [review]
Looks great @minkyngkm - good job!

**Pull request merged**